client/js/entities/entityManager.js

```diff
@@ -29,16 +32,48 @@ function EntityManager(scene, conn, world, clock) {
 		}
 	};
 
-	conn.on('entity-create', function (payload) {
+	function payloadToHistoryState(payload) {
+		var kind = payload.Kind;
+		var result = { time: null, data: null };
+		if (kind === EntityKindPlayer) {
+			var bioticState = Protocol.parseBioticState(payload.State);
+			result.time = bioticState.entityState.lastUpdated;
+			result.data = bioticState;
+		} else if (kind === EntityKindWorldItem) {
+			var worldItemState = Protocol.parseWorldItemState(payload.State);
+			result.time = worldItemState.entityState.lastUpdated;
+			result.data = worldItemState;
+		} else {
+			return null;
+		}
+		return result;
+	}
+
+	conn.on('entity-create', function(payload) {
 		var id = payload.ID;
+		var kind = payload.Kind;
+
 		if (controllers[id]) {
-			console.warn("Got entity-create message for entity which already exists!", id);
+			console.warn("Got an entity create message for entity which already exists!", id);
 			return;
 		}
-		var entity = EntityManager.makeEntity(payload);
+
+		var initialState = payloadToHistoryState(payload);
+		if (initialState == null) {
```

Comment:
I don't want to use !initialState, that implies something like "0" would cause this to run.
I won't use ===, because if it's undefined, I want it to run.
null == undefined: true
null === undefined: false